src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $card-width;
+  padding: $card-width * 0.01 * 16 $card-width * 0.01 * 8 $card-width * 0.01 * 8;
+  border: 1px solid $color-card-border;
+
+  @include transition-options(transform);
+
+  &__image {
+    width: $card-width * 0.8;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $font-medium;
+    margin-bottom: 4px;
+
+    @include transition-options(color);
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
```

Comment:
The selector '&:hover &__title' might not work as intended. Ensure that the hover effect is applied correctly to the title when the card is hovered. The space between '&:hover' and '&__title' indicates a descendant selector, which might not be what you want.